src/EnergyPlus/DataLoopNode.hh

```diff
@@ -447,7 +447,7 @@ namespace DataLoopNode {
         Real64 MassFlowRateMaxAvail = 0.0;              // {kg/s}
         Real64 MassFlowRateSetPoint = 0.0;              // {kg/s}
         Real64 Quality = 0.0;                           // {0.0-1.0 vapor fraction/percent}
-        Real64 Press = 0.0;                             // {Pa}
+        Real64 Press = 101325.0;                        // {Pa}
```

Comment:
I don't have a problem with this change. A default is a default. @mjwitte ?

Reply:
1. Yes, this is fine, but it should be `DataEnvironment::StdPressureSeaLevel;` instead of a hard value. 
2. Do we still need the constructors for this below?
https://github.com/NREL/EnergyPlus/blob/1ae086156a50480d51aaffc0642c82fad6b24aa2/src/EnergyPlus/DataLoopNode.hh#L491-L497
3. What I've run into in unit tests is this being zero, because it gets used heavily in Psychrometric calls.
https://github.com/NREL/EnergyPlus/blob/4d39dff4a6e899d6215b432714ee6cb6f6dad187/src/EnergyPlus/DataEnvironment.hh#L116
It would be great to initialize this to `DataEnvironment::StdPressureSeaLevel` if that doesn't cause any hiccups.

Reply:
First two comments addressed!
Because this PR is in response to a specific bug and only deals with NodeData, I'm going to hold off on #3 - just getting to here with single-line changes exposing multiple test and regression failures, I am cagey about changing anything outside of a very tiny scope. Happy to create another issue for it though.